scripts/deploy.py

```diff
@@ -38,6 +38,8 @@
 
 
 if __name__ == '__main__':
+    if os.environ.get('TRAVIS_SECURE_ENV_VARS', None) != 'true':
```

Comment:
pop(k) vs pop(k, None) always causes me to forget whether get(k, None) is optional or not. You're probably right that your ways is better though (but I didn't see this before I clicked merge)